### ISSUE TITLE
Error on translation import

### ISSUE THREAD

**Issue by mgruner** [zammad/zammad#4019]
### Infos:

* Used Zammad version: 5.1.0
* Installation method (source, package, ..): git
* Operating system: MacOS
* Database + version: PostgreSQL 14
* Elasticsearch version: --
* Browser + version: --

### Expected behavior:

* Zammad imports translations correctly from Weblate.

### Actual behavior:

* Zammad chokes on the very latest state from Weblate for zammad-stable:

```
 rake db:seed: rake aborted!
SimplePoParser::ParserError: SimplePoParser::ParserErrorSyntax error in lines
Syntax error in msgctxt
Syntax error in msgid
Syntax error in msgstr
Unexpected content after expected message end "\"跟进检"
Parseing result before error: '{:reference=>"db/seeds/settings.rb", :msgid=>["", "By default, the follow-up check is done via the subject of an email. This ", "setting lets you add more fields for which the follow-up check will be ", "executed."], :msgstr=>"默认情况下，通过电子邮件的主题进行跟进检查。这个设置允许你增加更多的字段执行"}'
SimplePoParser filtered backtrace: SimplePoParser::ParserError
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:179:in msgstr'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:134:inmsgid'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:114:in msgctxt'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:51:inlines'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:75:in comment'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:49:inlines'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:25:in parse'
    from /usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser.rb:19:inparse_message'

Caused by:
SimplePoParser::ParserError: Syntax error in lines
Syntax error in msgctxt
Syntax error in msgid
Syntax error in msgstr
Unexpected content after expected message end "\"跟进检"
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:179:in msgstr'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:134:inmsgid'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:114:in msgctxt'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:51:inlines'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:75:in comment'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:49:inlines'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser/parser.rb:25:in parse'
/usr/local/rvm/gems/ruby-2.7.4/gems/simple_po_parser-1.1.5/lib/simple_po_parser.rb:19:inparse_message'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser/tokenizer.rb:23:in parse_block'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser/tokenizer.rb:29:inblock in extract_entries'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser/tokenizer.rb:27:in each'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser/tokenizer.rb:27:inextract_entries'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser/tokenizer.rb:17:in extract'
/usr/local/rvm/gems/ruby-2.7.4/gems/PoParser-3.2.5/lib/poparser.rb:30:inparse_file'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:72:in block in strings_for_locale'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:70:ineach'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:70:in strings_for_locale'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:24:insync_locale_from_po'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:13:in block (2 levels) in sync'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/connection_adapters/abstract/database_statements.rb:280:inblock in transaction'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/connection_adapters/abstract/transaction.rb:280:in block in within_new_transaction'
/usr/local/rvm/gems/ruby-2.7.4/gems/activesupport-6.0.4.7/lib/active_support/concurrency/load_interlock_aware_monitor.rb:26:inblock (2 levels) in synchronize'
/usr/local/rvm/gems/ruby-2.7.4/gems/activesupport-6.0.4.7/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in handle_interrupt'
/usr/local/rvm/gems/ruby-2.7.4/gems/activesupport-6.0.4.7/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:inblock in synchronize'
/usr/local/rvm/gems/ruby-2.7.4/gems/activesupport-6.0.4.7/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in handle_interrupt'
/usr/local/rvm/gems/ruby-2.7.4/gems/activesupport-6.0.4.7/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:insynchronize'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/connection_adapters/abstract/transaction.rb:278:in within_new_transaction'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/connection_adapters/abstract/database_statements.rb:280:intransaction'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/transactions.rb:212:in transaction'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:12:inblock in sync'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/relation/delegation.rb:88:in each'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/relation/delegation.rb:88:ineach'
/home/zp_00018844/zammad/app/models/translation/synchronizes_from_po.rb:11:in sync'
/home/zp_00018844/zammad/db/seeds.rb:36:in<main>'
/usr/local/rvm/gems/ruby-2.7.4/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:in load'
/usr/local/rvm/gems/ruby-2.7.4/gems/bootsnap-1.11.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:39:inload'
/usr/local/rvm/gems/ruby-2.7.4/gems/railties-6.0.4.7/lib/rails/engine.rb:559:in load_seed'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/tasks/database_tasks.rb:440:inload_seed'
/usr/local/rvm/gems/ruby-2.7.4/gems/activerecord-6.0.4.7/lib/active_record/railties/databases.rake:331:in block (2 levels) in <main>'
/usr/local/rvm/gems/ruby-2.7.4/gems/rake-13.0.6/exe/rake:27:in<top (required)>'
/usr/local/rvm/gems/ruby-2.7.4/bin/ruby_executable_hooks:22:in eval'
/usr/local/rvm/gems/ruby-2.7.4/bin/ruby_executable_hooks:22:in<main>'
[...]
```

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by mgruner** [comment]
Probably related to https://github.com/experteer/simple_po_parser/issues/3

**Comment by mgruner** [comment]
TODO:
- Fix the actual issue
- Extend tests to cover importing all languages

**Comment by mgruner** [comment]
Fixed by f111d0cce3aeea5fac5c554ebcaea0607e49d5e7.
